hikari/commands.py

```diff
@@ -146,6 +146,18 @@ class CommandOption:
     If `builtins.None`, then all channel types will be accepted.
     """
 
+    min_value: typing.Union[int, float, None] = attr.field(default=None, repr=False)
+    """The minimum value permitted (inclusive)
+
+    If the option type is `hikari.commands.OptionType.INTEGER` than this must be an `builtins.int`.
+    """
+
+    max_value: typing.Union[int, float, None] = attr.field(default=None, repr=False)
+    """The minimum value permitted (inclusive)
+
+    If the option type is `hikari.commands.OptionType.INTEGER` than this must be an `builtins.int`.
+    """
```

Comment:
```suggestion
    min_value: typing.Union[int, float, None] = attr.field(default=None, repr=False)
    """The minimum value permitted (inclusive).

    This will be `builtins.int` if the type of the option is `hikari.commands.OptionType.INTEGER`
    and `builtins.float` if the type is `hikari.commands.OptionType.NUMBER`. 
    """

    max_value: typing.Union[int, float, None] = attr.field(default=None, repr=False)
    """The maximum value permitted (inclusive).

    This will be `builtins.int` if the type of the option is `hikari.commands.OptionType.INTEGER`
    and `builtins.float` if the type is `hikari.commands.OptionType.NUMBER`. 
    """
```